resources/msgBusMetricsMsg.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
file name still wrong

Reply:
The name is right. I renamed it to be plural (from msgBusMetricMsg) as it can hold more metrics in the future. But you have a point about the naming. I'll rename the rest too.